docs/reference/tools-reference/open_source_llm-tool.md

```diff
@@ -0,0 +1,60 @@
+# Open SourceLLM
```

Comment:
Maybe we can have two different PRs for the two different tools?

Reply:
Good point - here are the new PR's:
#370
#371